cpp/foxglove-websocket/src/serialization.cpp

```diff
@@ -103,17 +103,63 @@ void to_json(nlohmann::json& j, const Service& service) {
     {"id", service.id},
     {"name", service.name},
     {"type", service.type},
-    {"requestSchema", service.requestSchema},
-    {"responseSchema", service.responseSchema},
   };
+
+  if (service.request.has_value()) {
+    j["request"] = service.request.value();
+  }
```

Comment:
nit: possible cleanup
```suggestion
  if (service.request) {
    j["request"] = *service.request;
  }
```